static/app/views/profiling/landing/slowestFunctionsTable.tsx

```diff
@@ -238,6 +259,29 @@ function SlowestFunctionsProjectBadge(props: SlowestFunctionsProjectBadgeProps)
   ) : null;
 }
 
+interface SlowestFunctionTimeSeriesProps {
+  function: Profiling.FunctionMetric;
+}
+
+function SlowestFunctionTimeSeries(props: SlowestFunctionTimeSeriesProps) {
+  const projects = useMemo(() => {
+    return props.function.examples
+      .map(f => {
+        if (typeof f !== 'string' && 'project_id' in f) return f.project_id;
+        return null;
+      })
+      .filter(n => n !== null);
+  }, [props.function]);
```

Comment:
Nit: you may want to dedupe project ids from this list

Reply:
Ah yes, right. Let me do that